nixos/modules/virtualisation/hcloud-config.nix

```diff
@@ -0,0 +1,131 @@
+# Configuration for Hetzner Cloud virtual servers.
+
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.hcloud;
+  dynamicHostname = config.networking.hostName == "";
+in
+
+{
+  imports = [
+    ../profiles/qemu-guest.nix
+  ];
+
+  options = {
+    hcloud = {
+      efi = lib.mkOption {
+        default = pkgs.stdenv.hostPlatform.isAarch64;
+        defaultText = lib.literalExpression "pkgs.stdenv.hostPlatform.isAarch64";
+        internal = true;
```

Comment:
x86_64-linux can't use efi?

Reply:
Unfortunately not. They only seem to do legacy bios boot.